### PR TITLE
Bpf 1645

### PR DESCRIPTION
**Pull request by harlyon**
#### What is the purpose of this pull request?
https://tfginfotec.atlassian.net/browse/BPF-1645

#### Screenshots or example usage
Screenshot 2023-09-14 at 17 49 10

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/FurnitureNote.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const handles = useCssHandles(CSS_HANDLES)
+
+  return (
+    <div className={`${handles.furnitureNote}`}>
+      <h4 className={`${handles.noteTitle}`}>furniture delivery</h4>
+      <p>
+        {shippingMethod === 'deliver' ? deliveryMessage : collectMessage}
+        <br />{' '}
+      </p>
+      <a href="#/customer-service/help" className={handles.noteLink}>
```
Comment (simmbiote) [inline react/FurnitureNote.tsx:38]: Is this link correct? Shouldn't it be somewhere on help.bash.com?

File: react/ConfirmationMessage.tsx [hunk 2]
```diff
@@ -3,43 +3,57 @@ import { FormattedMessage } from 'react-intl'
...
+            }}
+          />
+        ) : (
+          <FormattedMessage
+            id="store/header.phone"
+            values={{
+              lineBreak: <br />,
+              phone: (
+                <strong className="nowrap">
+                  {orderGroup.orders[0].clientProfileData.phone}
```
Comment (simmbiote) [inline react/ConfirmationMessage.tsx:48]: This could be undefined

File: react/ConfirmationMessage.tsx [hunk 3]
```diff
@@ -3,43 +3,57 @@ import { FormattedMessage } from 'react-intl'
...
+import { FurnitureNote } from './FurnitureNote'
 
 const CSS_HANDLES = ['confirmationMessage']
 
 const ConfirmationMessage: FC = () => {
   const handles = useCssHandles(CSS_HANDLES)
   const orderGroup = useOrderGroup()
 
+  const shippingMethod =
+    orderGroup.orders[0].pickUpParcels.length > 0 ? 'collect' : 'deliver'
```
Comment (simmbiote) [inline react/ConfirmationMessage.tsx:15]: Is there risk of pickUpParcels being undefined?

**Review by simmbiote — CHANGES_REQUESTED** [review]

File: react/ConfirmationMessage.tsx (hunk 3 above)
Comment (harlyon) [reply react/ConfirmationMessage.tsx:15]: @simmbiote these are not my changes oops

**Review by simmbiote — APPROVED** [review]

**Review by simmbiote — APPROVED** [review]

**Review by lezegom — APPROVED** [review]

**Pull request merged**